src/Console/Command/ProvidersCommand.php

```
@@ -0,0 +1,41 @@
+ * ```
+ *
+ * @author Wilmer Arambula <terabytesoftw@gmail.com>
+ * @since 0.1
+ */
+#[AsCommand(
+    name: 'providers',
+    description: "Lists all scaffold providers recorded in 'scaffold-lock.json' with their file counts.",
+)]
+final class ProvidersCommand extends AbstractScaffoldCommand
+{
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$input'. (undefined)

(UnusedFormalParameter)